src/newcharacter.cpp

```diff
@@ -885,6 +885,12 @@ void Character::initialize( bool learn_recipes )
         add_proficiency( pri );
     }
 
+    // Add profession recipes
+    for( const recipe_id id : prof->recipes() ) {
+        const recipe &r = recipe_dictionary::get_craft( id->result() );
+        learn_recipe( &r );
```

Comment:
Nitpick, but since `recipe` is used only once, you may avoid its explicit declaration and put its value directly inside `learn_recipe`.

Reply:
I tried that originally with `learn_recipe( &( recipe_dictionary::get_craft( id->result() ) ) );` but Clang threw a fit because it doesn't like using `&( )` inline for whatever reason. So I had change it.